modules/standards/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@phenyl/standards",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "upbin tsc --declaration",
+    "type-check": "upbin tsc --noEmit",
+    "test": "yarn test:ts && yarn test:mocha",
+    "test:mocha": "NODE_ENV=test nyc --nycrc-path .nycrc mocha --require test/helper/babel-register.js --recursive test/spec/**/*.ts --color always",
+    "test:ts": "upbin tsc -p tsconfig.test.json"
+  },
+  "files": [
+    "src",
+    "dist",
+    "package.json",
+    "jsnext.js",
+    "LICENSE",
+    "README.md",
+    "decls"
+  ],
+  "devDependencies": {
+    "@types/mocha": "^5.0.5",
+    "@babel/register": "^7.0.0",
+    "mocha": "^5.0.5",
+    "nyc": "^11.6.0",
+    "power-assert": "^1.4.4",
+    "upbin": "^0.8.1"
+  },
+  "dependencies": {
+    "@phenyl/interfaces": "*",
+    "@phenyl/utils": "*",
+    "@sp2/format": "~1.2.3",
+    "@sp2/updater": "~1.1.0",
+    "mongolike-operations": "~0.1.5",
+    "oad-utils": "~0.3.0",
+    "phenyl-interfaces": "~0.4.2",
+    "phenyl-utils": "~0.4.2",
+    "power-assign": "~0.2.10",
+    "power-crypt": "~0.1.4"
+  },
+  "author": "Shin Suzuki <shinout310@gmail.com>",
+  "repository": "https://github.com/phenyl-js/phenyl/tree/master/modules/phenyl-standards",
+  "license": "Apache License 2.0"
+}
```

Comment:
Could you reorder the keys in `package.json` so that it matches with other modules? (See package.json in `interfaces` for example)

Reply:
Sure, Fixed it